backend-api/app/routers/system.py

```
@@ -321,32 +290,196 @@ async def run_async(cmd: list[str], timeout: int = 6):
+async def ffmpeg_update():
+    """Download and install the latest Jellyfin FFmpeg package.
+
+    This replaces the FFmpeg binaries in-place inside the running container.
+    A container restart is recommended afterwards.
+    """
+    latest = _get_latest_jellyfin_ffmpeg()
+    if "error" in latest:
+        raise HTTPException(status_code=502, detail=f"Cannot reach Jellyfin repo: {latest['error']}")
+
+    url = latest["url"]
+    deb_path = "/tmp/jellyfin-ffmpeg-update.deb"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/jellyfin-ffmpeg-update.deb"

(S108)